installers/msi-language/StateCommandExists/CustomAction.cs

```diff
@@ -9,7 +9,7 @@ public class CustomActions
         [CustomAction]
         public static ActionResult CheckCommands(Session session)
         {
-            ActiveState.RollbarHelper.ConfigureRollbarSingleton(session["COMMIT_ID"]);
+            ActiveState.RollbarHelper.ConfigureRollbarSingleton(session["MSI_VERSION"]);
```

Comment:
I'm calling this as the existing code does, but it doesn't match `session.CustomActionData` in other files. Does this need to be different?

Reply:
@MDrakos or @mdrohmann Would you, please, shed light on why this call is different than other session usages?

Reply:
This custom action is scheduled as `immediate` so it has access to the session object. The custom actions that are deferred do not, so we must pass the data it needs via the `CustomActionData`. This is all done in the `Product.p.wxs` file.